openlibrary/macros/EditionNavBar.html

```diff
@@ -32,7 +32,7 @@
     </li>
   </ul>
   $if not desktop_only:
-    <span class="nav-arrow">
-      <img src="/static/images/nav-arrow.svg" role="img">
-    </span>
+  <button class="nav-arrow" title="$_('Go to next section')">
+    <img src="/static/images/nav-arrow.svg" alt="">
+  </button>
```

Comment:
```suggestion
    <button class="nav-arrow" title="$_('Go to next section')">
      <img src="/static/images/nav-arrow.svg" alt="">
    </button>
```
Indentation level of this code block should remain the same.

Reply:
Fix styling of button:
![image](https://github.com/user-attachments/assets/3d09a19d-f0fb-434e-9b52-bcb6820769b4)